### ISSUE TITLE
JSON value '<null>' of type NSNull cannot be converted to NSNumber

### ISSUE THREAD

**Issue by dantman** [react-native-community/react-native-svg#508]
I upgraded to the latest react-native and github master version of react-native-svg and now I'm getting this error.

Here is a demo app that reproduces the bug:
https://github.com/dantman/rnsvg-bug/blob/master/App.js

![simulator screen shot - iphone x - 2017-11-13 at 18 15 41](https://user-images.githubusercontent.com/53399/32759554-dedda43a-c89e-11e7-980b-8ecf61abcb6c.png)
```
JSON value '<null>' of type NSNull cannot be converted to NSNumber

+[RCTConvert NSNumber:]
    RCTConvert.m:59
__RCTConvertArrayValue_block_invoke
-[__NSArrayI enumerateObjectsWithOptions:usingBlock:]
RCTConvertArrayValue
+[RCTConvert NSNumberArray:]
+[RCTConvert UIColor:]
+[RCTConvert CGColor:]
+[RCTConvert(RNSVG) RNSVGCGColor:offset:]
-[RNSVGSolidColorBrush initWithArray:]
+[RCTConvert(RNSVG) RNSVGBrush:]
__49-[RCTComponentData createPropBlock:isShadowView:]_block_invoke.163
__49-[RCTComponentData createPropBlock:isShadowView:]_block_invoke.167
__49-[RCTComponentData propBlockForKey:isShadowView:]_block_invoke_2
RCTPerformBlockWithLogFunction
RCTPerformBlockWithLogPrefix
__49-[RCTComponentData propBlockForKey:isShadowView:]_block_invoke
__37-[RCTComponentData setProps:forView:]_block_invoke
-[__NSDictionaryM enumerateKeysAndObjectsWithOptions:usingBlock:]
-[RCTComponentData setProps:forView:]
__50-[RCTUIManager createView:viewName:rootTag:props:]_block_invoke
__RCTExecuteOnMainQueue_block_invoke
_dispatch_call_block_and_release
_dispatch_client_callout
_dispatch_main_queue_callback_4CF
__CFRUNLOOP_IS_SERVICING_THE_MAIN_DISPATCH_QUEUE__
__CFRunLoopRun
CFRunLoopRunSpecific
GSEventRunModal
UIApplicationMain
main
start
```

**Comment by Tennen** [comment]
+1

**Comment by dantman** [comment]
Should be fixed by #512.

**Comment by ayugohel** [comment]
+1
